setup.py

```diff
@@ -58,13 +62,25 @@
 import shutil
 import sys
 
+import jupyterlab
```

Comment:
Please add try-catch for this import. The script should not crash when building machine has not installed jupyter.
And I personally suggest put this and `jupyter_lab_version = ...` into a function `get_jupyter_lab_version()`.

Reply:
changed, In this way, the building machine without jupyterlab installed will report error and the package related jupyter lab extension will not built, but continue to build other package